ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -170,7 +151,7 @@
   [request]
   (let [res (prepare-generic-document request)
         headers (:headers request)
-        {:keys [spec-key spec-version provider-id native-id request-context concept]} res
+        {:keys [spec-key spec-version _provider-id _native-id request-context concept]} res
```

Comment:
If we don't need provider-id or native-id anymore as keys, they should be deleted.